phpBB/phpbb/db/migration/data/v33x/extend_bbcode_tooltip.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v33x;
+
+class extend_bbcode_tooltip extends \phpbb\db\migration\migration
+{
+	public function effectively_installed()
+	{
+		return !$this->db_tools->sql_column_exists($this->table_prefix . 'bbcodes', 'bbcode_helpline');
```

Comment:
I actually wonder if there would be a way to find out which type the column is. In that case, this would then also return true of the helpline is of type `TEXT_UNI`

Reply:
I do not think there is something ready to be used for that, IIRC.
If somehow that column is already of that type nothing changes, just it gets changed one more time to the same type. This only could happen if someone manually changed that type though.

Reply:
Probably using a query which uses `SELECT DATA_TYPE, COLUMN_NAME  FROM etc..`, but as I said there is nothing AFAIR in the db tools for that.